xml/deployment_hwrecommend.xml

```diff
@@ -378,24 +378,30 @@
  </sect1>
  <sect1 xml:id="ceph-install-ceph-deploy-network">
   <title>Network Recommendations</title>
-
   <para>
-   For the &ceph; cluster network environment, we recommend two bonded 25 GbE
-   network interfaces. The network interfaces should be split into a public
-   client part, and a trusted internal part using VLANs. Use the 802.3ad
-   bonding mode to provide maximum bandwidth and resiliency. Find more details
-   about configuring bonded network interfaces in
-   <link xlink:href="https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-network.html#sec-network-iface-bonding"/>
+   For the &ceph; cluster network environment, we recommend two bonded 25 GbE (or faster)
+   network interfaces bonded using 802.3ad (LACP). The use of two network interfaces
+   provides aggregation and fault-tolerance. The bond should then be used to provide
+   two VLAN interfaces, one for the public network, and the second for the cluster network.
+   Details on bonding the interfaces can be found in
+   <link xlink:href="https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-network.html#sec-network-iface-bonding"/>.
   </para>
-
   <para>
-   The public VLAN serves to provide the service to the customers, while the
-   internal part provides for the authenticated &ceph; network communication.
-   The main reason for this is that although &ceph; provides authentication and
-   protection against attacks once secret keys are in place, the messages used
-   to configure these keys may be transferred openly and are vulnerable.
+    Enhance fault tolerance through isolating the components into failure domains.
+    To improve fault tolerance of the network, bonding one interface from two
+    separate Network Interface Cards (NIC) offers protection against failure
+    of a single NIC. Similarly, creating a bond across two switches protects
+    against failure of a switch. We recommend consulting with the network
+    equipment vendor in order to architect the level of fault tolerance required.
+  </para>
+  <para>
+    The cluster utilizes two networks. The cluster network is used exclusively
+    for private backend communication, while the public address is used for
```

Comment:
we talk about 2 networks. The cluster "network" and then here you use the public "address". I assume address here should be network too.

Reply:
correct.